### DISCUSSION TITLE
I have been verifed as null

### DISCUSSION THREAD

**Discussion by lucaboesch** [community/community#110303] [GitHub Education]
### Hi everyone,

Yesterday, I have submitted a discount request for education as teacher on https://education.github.com/discount_requests/application. It probably didn't go through properly as I can tell from the message now showing:

Hi, <username>! You were last verified as a null on Feb 26, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

I'm now officially veryfied as null!

**Comment by Lautaro-Aguilar** [comment]
Same here

**Comment by adarsh-site** [comment]
It seems there might have been an issue with your discount request, as the message indicates that you were last verified as "null" on February 26, 2024. This could mean that your verification status was not properly updated or that there was some error in the process.

To address this, I recommend reaching out to GitHub Education support directly. You can do this by visiting the GitHub Education website and looking for a support or contact page. Explain the situation to them and provide any relevant details about your discount request. They should be able to assist you in resolving the issue and ensuring that your educator status is properly verified, allowing you to access the academic benefits you're entitled to.

**Comment by RkBeau** [comment]
I got the same message after submitting my id 3 times being told it did not have the school name on it which it clearly does
